drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/Docks.md

```diff
@@ -0,0 +1,28 @@
+Details on Docks management inside ScenarioSimulationEditor
+===========================================================
```

Comment:
Using md-file makes sense, just need to make sure the i18n team knows about it.

Reply:
Ah. Reading this out now, it is explanation about how the docks work, not show to users :)
But perhaps a good idea for presenting the docks help to users too.